Sources/SpeziBluetooth/CoreBluetooth/BluetoothPeripheral.swift

```diff
@@ -207,17 +214,39 @@ public class BluetoothPeripheral { // swiftlint:disable:this type_body_length
     /// Disconnect the ongoing connection to the peripheral.
     ///
     /// Cancels an active or pending connection to a peripheral.
+    @available(*, deprecated, message: "Please migrate to the async throwing version of disconnect().")
```

Comment:
It is currently not throwing; we might need to adjust this warning or actually make it throwing.

Reply:
Ups, yeah it's just async now, which, shouldn't a huge issue, as most call sites would use await anyways due to actor isolation

Reply:
Sounds good 🚀